src/cutpruning.jl

```diff
@@ -15,39 +38,56 @@ Exact pruning of all polyhedral functions in input array.
 * `params::SDDPparameters`:
 * `Vector{PolyhedralFunction}`:
     Polyhedral functions where cuts will be removed
-* `iteration_count::Int64`:
+* `trajectories::Array{Float64, 3}`
+    Previous trajectories
+* `territory::Array{Territories}`
+    Container storing the territory for each cuts
+* `it::Int64`:
     current iteration number
 * `verbose::Int64`
 """
 function prune_cuts!(model::SPModel,
```

Comment:
I think that the function should be splitted with a which param.pruning[:type] calling a dedicated function.